client/components/plans/plan-price/index.jsx

```diff
@@ -20,6 +25,11 @@ module.exports = React.createClass( {
 				return this.translate( 'Free', { context: 'Zero cost product price' } );
 			}
 
+			if ( abtest( 'monthlyPlanPricing' ) === 'monthly' && this.props.isInSignup ) {
+				const monthlyPrice = +( rawPrice / 12 ).toFixed( 2 );
+				formattedPrice = formattedPrice.replace( rawPrice, monthlyPrice );
```

Comment:
Is this going to work for prices formatted with different separators? The formatted price should be localized and in some countries the decimal separator might be a comma instead of a dot. This might not be an issue for this A/B test since it will run only for English users but that's something we should keep in mind.